### DISCUSSION TITLE
Inflation Factor Computation on EEIO

### DISCUSSION THREAD

**Discussion by Wab1l** [USEPA/supply-chain-factors#86] [Q&A]
As the EEIO V1.3 (The latest) are for the amount spend in $ for the year 2022, so if i want to calculate emissions for my organizations reporting year end which is June 2023, how do i calculate inflation factor?
Should I calculate it from 1 January 2022 to 1 January 2023 or
Should I calculate it from 1 January 2022 to 30 June 2023 or 
Should I calculate it from 31 December 2022 to 30 June 2023

**Comment by bl-young** [comment]
We recommend using commodity specific inflation factors when available. These typically are delayed by a year or two from BEA. We don't have specific recommendations on how adjust for 2023 spend. See for example https://github.com/USEPA/supply-chain-factors/discussions/68
